src/vs/workbench/contrib/chat/browser/actions/chatActions.ts

```diff
@@ -88,7 +89,7 @@ class OpenChatGlobalAction extends Action2 {
 		});
 	}
 
-	override async run(accessor: ServicesAccessor, opts?: string | IChatViewOpenOptions): Promise<void> {
+	override async run(accessor: ServicesAccessor, opts?: string | IChatViewOpenOptions, variables?: IChatRequestVariableEntry[]): Promise<void> {
```

Comment:
Add it to `IChatViewOpenOptions`, but we will call it a temporary thing until screenshots are managed by vscode, `IChatRequestVariableEntry` has too much going on to be a proper argument command passed by an extension IMO.